src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -38,6 +38,7 @@ public class Result extends DomainObject {
 
     @Column(name = "result_string")
     @JsonView(QuizView.After.class)
+    @Deprecated
     private String resultString;
```

Comment:
```suggestion
    // IMPORTANT on removal:
    // Mention in release notes that an intermediary upgrade to version 5.8.5 or later is necessary before updating to the current version, as otherwise the database migration can no longer convert the old data into the new representation.
    @Column(name = "result_string")
    @JsonView(QuizView.After.class)
    @Deprecated
    private String resultString;
```